src/view.js

```diff
@@ -171,9 +167,9 @@ const View = Backbone.View.extend({
   // object literal, or a function that returns an object
   // literal. All methods and attributes from this object
   // are copies to the object passed in.
-  mixinTemplateContext(target = {}) {
+  mixinTemplateContext(target) {
     const templateContext = _.result(this, 'templateContext');
-    return _.extend(target, templateContext);
+    return _.extend({}, target, templateContext);
```

Comment:
I'm in favour of making this quicker and I'm not convinced anyone would have modified this much (if at all). _However_ it is breaking and we do commit to semver.